pixel_clusterizer/clusterizer.py

```diff
@@ -9,11 +9,25 @@
                           ('charge', '<u2')])
 
 
+def np_uint_type_chooser(number):
```

Comment:
They are slow and the function should be provided with index arrays that have constant lookup time ([Space <-> Time tradeoff](https://en.wikipedia.org/wiki/Space%E2%80%93time_tradeoff)].
Thus I would delete.